package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://github.com/zambezi/caballo-vivo#readme",
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": "^17.0.0"
```

Comment:
Shouldn't it be a ranged version allowing 16 and 17?

Reply:
Actually it should be an open-ended ranged version, like `>=16.8.0`. We want to support everything from the version that supports hooks.

Reply:
I think that's what we need, yes.